async-query-core/src/main/java/org/opensearch/sql/asyncquery/Dummy.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.asyncquery;
+
+// This is a dummy class for scaffolding and should be deleted later
+public class Dummy {
```

Comment:
Do we really need a commit for this?

Reply:
If we move something, the build would break since something is depending on that. There might be a better way to verify the dependency works, but I thought it is the simplest way.

Reply:
I am ok even if we don't verify if the dependency works in the first PR and we can verify when the dependency actually comes out.